Improve wording (#2472)
hammerandchisel/discord-api-docs: docs/game_sdk/SDK_Starter_Guide.md

- [Discord Game SDK](https://dl-game-sdk.discordapp.net/2.5.6/discord_game_sdk.zip)

There are a few things in there, but let's quickly talk about what the SDK actually _is_. Inside the `lib/` folder, you'll see `x86/` and `x86_64/` that have some `.lib`, `.bundle`, and `.dll` files. These are the things you want to distribute with your game.

These files are comprised of two parts: a "stub", and fallback modules. What that means is that when everything is running smoothly, the DLLs will just call back to the local running Discord client to do the heavy lifting. If, however, something is wrong, like a breaking change, the files also include "fallback" modules that reflect the native SDK modules in Discord at the time that version of the SDK was published. TLDR - you don't need to worry about breaking changes.

## Get Set Up

Next, we need to set up the application for your game. An application is the base "entity" in Discord for your game; it's what all the builds, branches, SKUs, store pages, assets, etc., will be filed under.

> Head over to our [developer site](https://discord.com/developers/) and create an account/login if you haven't yet. The first thing we're going to do is create a Team. Teams are groups of developers working together on applications; you should create a team for your organization at [https://discord.com/developers/teams](https://discord.com/developers/teams). You can invite other users to join your team and work on applications together with you.

Author: ```suggestion
Head over to our [developer site](https://discord.com/developers/) and create an account/log in if you haven't yet. The first thing we're going to do is create a Team. Teams are groups of developers working together on applications; you should create a team for your organization at [https://discord.com/developers/teams](https://discord.com/developers/teams). You can invite other users to join your team and work on applications together with you.
```

> Now that your team is created, you'll want to make an application. To do so, click on "Applications" at the top of the page and create an application. Make sure you pick your newly-created team in the `Team` dropdown. You want your team to own the application; this unlocks store functionality! Now that your app is made let's dive into some more setup.

Author: ```suggestion
Now that your team is created, you'll want to make an application. To do so, click on "Applications" at the top of the page and create an application. Make sure you pick your newly-created team in the `Team` dropdown. You want your team to own the application; this unlocks store functionality! Now that your app is made, let's dive into some more setup.
```

> warn
> If you're integrating our SDK into an already-released game, there's a good chance that we may _already have_ an application in our database for your game! Reach out to our [Dev Support](https://dis.gd/devsupport) to learn more

First, we'll need to set an OAuth2 redirect URL. You can add `http://127.0.0.1` in there for now; this powers some behind-the-scenes stuff you don't need to worry about.

> Next, copy the **Client ID** at the top of the page. This id also referred to as an "application id", is your game's unique identifier across Discord. Keep it handy!

Author: ```suggestion
Next, copy the **Client ID** at the top of the page. This id, also referred to as an "application id", is your game's unique identifier across Discord. Keep it handy!
```

First, you'll want to copy the header files and `.cpp` files to a folder somewhere in your project directory. For ease of a quickstart example, you can put them right inside your `Source/your-project-name` folder; I'd put them in a containing folder called something like `discord-files/`.

Second, you'll want to copy the `.dll` and `.lib` files from the `lib/x86_64` folder of the downloaded zip. These files should be put in `your-project-name/Binaries/Win64/`. For win32, take the files from `x86/` and put them in `your-project-name/Binaries/Win32`.

Next, we need to link these files within our project so that we can reference them. If you open up your project's `.sln` file in Visual Studio, you'll find a file called `your-project-name.Build.cs`. We're going to add the following lines of code to that file:

```cpp
/*

>     ABSOLUTE_PATH_TO_DISCORD_FILES_DIRECTORY will look something like this:

Author: ```suggestion
    ABSOLUTE_PATH_TO_DISCORD_FILES_DIRECTORY will look something like:
```

PublicIncludePaths.Add(ABSOLUTE_PATH_TO_DISCORD_FILES_DIRECTORY)

/*

>     ABSOLUTE_PATH_TO_LIB_FILE will look something like this:

Author: ```suggestion
    ABSOLUTE_PATH_TO_LIB_FILE will look something like:
```

        2. Open Discord
        3. Attempt to re-open your game
        Step 3 will fail when running directly from the Unreal Engine editor

>         Therefore, always keep Discord running during tests or use Discord.CreateFlags.NoRequireDiscord

Author: ```suggestion
        Therefore, always keep Discord running during tests, or use Discord.CreateFlags.NoRequireDiscord
```

I think the rules would say no, but I agree that this sounds better when reading

1. Download Discord Canary. This is our most updated build, and is good to develop against: [Windows](https://discord.com/api/download/canary?platform=win) - [Mac](https://discord.com/api/download/canary?platform=osx)
2. Download a second Discord Build. Here's our Public Test Build: [Windows](https://discord.com/api/download/ptb?platform=win) - [Mac](https://discord.com/api/download/ptb?platform=osx)
3. Open up two Discord clients. We recommend you develop against Discord Canary, so you can use PTB or Stable for your test account

> 4. Login with two separate users. Make sure any test account is added to the application's App Whitelist in the portal!

Author: ```suggestion
4. Log in with two separate users. Make sure any test account is added to the application's App Whitelist in the portal!
```
